extensions/blocks/videopress/editor.js

```diff
@@ -14,6 +16,33 @@ import withVideoPressEdit from './edit';
 import withVideoPressSave from './save';
 import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
 import deprecatedV1 from './deprecated/v1';
+import wrapPaidBlock from '../../shared/wrap-paid-block';
+import isSimpleSite from '../../shared/site-type-utils';
```

Comment:
this isn't a default export so should be `import { isSimpleSite } from '../../shared/site-type-utils';

Reply:
Done, thanks!